app/workers/historical_data_deletion_worker.rb

```diff
@@ -15,7 +15,7 @@ def perform
     delete_and_log("subscriber lists") { empty_subscriber_lists }
 
     # restricts deletion if emails are present
-    delete_and_log("subscribers") { Subscriber.where("deactivated_at < ?", max_retention_period) }
+    delete_and_log("subscribers") { historic_nullified_subscribers }
```

Comment:
```suggestion
    delete_and_log("subscribers") { historic_former_subscribers }
```

(This is no longer associated with nullification.)

Reply:
Good spot - I've just used `historic_subscribers`